src/servers/Cargo.toml

```
@@ -51,6 +51,7 @@ futures = "0.3"
 hashbrown = "0.14"
 headers = "0.3"
 hostname = "0.3.1"
+http = "0.2.12"
```

According to our code style, we don't set the minor version explicitly.
```suggestion
hostname = "0.3"
http = "0.2"
```